woocommerce/woocommerce-android
# Removes coupons outdated banner as well as its associated survey

Closes: #12956 


### Description

Coupons feature has been released for more than a year now (close to 2 years). The banner with the feature announcement and the link to the survey felt outdated and didn't make sense anymore. We decided to remove it. 

This PR does just that. Removes the banner and cleans the code for any reference related to the banner or the coupons survey. 

### Testing information
1. Make a clean install of the app. 
2. Open More Menu > Coupons section
3. Check the banner is not displayed anymore and that coupons list is displayed correctly. 

### The tests that have been performed

The above. Additionally I tested that the empty case for coupons is also displayed correctly. 


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
